scripts/generator/graphql_generator/csharp/SDK/ErrorFormatter.cs.erb

```diff
@@ -0,0 +1,22 @@
+namespace <%= namespace %> {
+    using <%= namespace %>.SDK;
+
+    public static class ErrorFormatter {
```

Comment:
I felt like transforming the errors from the response into a ShopifyError doesn't make sense inside the `TopLevelResponse` since generating the `ShopifyError` seems like an abstraction on top of it. Not sure if `ErrorFormatter` makes sense though.